spec/models/concerns/json_file_spec.rb

```diff
@@ -2,15 +2,21 @@
 require "rails_helper"
 
 RSpec.describe JsonFile, prep_metadata_sources: true do
-  let(:parent_object) { FactoryBot.create(:parent_object, oid: '16685691') }
-  let(:path_to_example_file) { Rails.root.join("spec", "fixtures", "ladybird", "16685691.json") }
+  let(:parent_object) { FactoryBot.create(:parent_object, oid: '2003431') }
+  let(:path_to_example_file) { Rails.root.join("spec", "fixtures", "ladybird", "2003431.json") }
+  around do |example|
+    perform_enqueued_jobs do
+      example.run
+    end
+  end
+
   before do
-    stub_metadata_cloud("16685691")
+    stub_metadata_cloud("2003431")
```

Comment:
This example has over 1,000 child objects - let's change it back to `16685691` (I'm sure this was a merge conflict)

Reply:
Fixed